docker-cloud/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# syntax=docker/dockerfile:1
+
+## Build
+FROM golang:1.19-bullseye as build
+
+WORKDIR /app
+
+COPY go.mod .
+COPY go.sum .
+
+RUN go mod download
+
+COPY *.go ./
```

Comment:
Intentionally different from the `.` destination of the above copies?

Reply:
Yes. Will comment this up